LDAP-Auth/Config/PluginConfiguration.cs

```diff
@@ -150,5 +167,59 @@ public PluginConfiguration()
         /// Gets or sets the password reset url.
         /// </summary>
         public string PasswordResetUrl { get; set; }
+
+        /// <summary>
+        /// Adds a user to the ldap users.
+        /// </summary>
+        /// <param name="userGuid">The user Guid.</param>
+        /// <param name="ldapUid">The LDAP UID associated with the user.</param>
+        public void AddUser(Guid userGuid, string ldapUid)
+        {
+            // Ensure we do not have more than one entry for a given user
+            // This may happen if a user tries to authenticate after their
+            // ldapUid has changed or if their Jellyfin account has been deleted
+            RemoveUser(userGuid);
+            RemoveUser(ldapUid);
+
+            var ldapUsers = LdapUsers.ToList();
+            var ldapUser = new LdapUser
+            {
+                LinkedJfUserId = userGuid,
+                LdapUid = ldapUid
+            };
+            ldapUsers.Add(ldapUser);
+            LdapUsers = ldapUsers.ToArray();
+        }
+
+        /// <summary>
+        /// Removes a user from the LDAP users.
+        /// </summary>
+        /// <param name="userGuid">The user id.</param>
+        public void RemoveUser(Guid userGuid)
+        {
+            var ldapUsers = LdapUsers.ToList();
+            ldapUsers.RemoveAll(user => user.LinkedJfUserId == userGuid);
+            LdapUsers = ldapUsers.ToArray();
+        }
+
+        /// <summary>
+        /// Removes a user from the LDAP users.
+        /// </summary>
+        /// <param name="ldapUid">The LDAP uid of the user.</param>
+        public void RemoveUser(string ldapUid)
```

Comment:
```suggestion
        private void RemoveUser(string ldapUid)
```

Reply:
I left that public in case it is needed for some reason in the future. Would you be happy if I leave it like that?